powerquery-docs/connectors/solarwinds-service-desk.md

```diff
@@ -34,41 +32,114 @@ Before you can use this connector to get SolarWinds Service Desk data, you must
 
 ## Connect to SolarWinds Service Desk
 
-To connect to SolarWinds Service Desk:
+The instructions below contain special steps for customers who previously used the BETA SWSD Power BI Connector to create reports and wish to connect those reports with the new out-of-the-box version.
 
-1. In Power BI Desktop, select **Get Data** from **Home**. Select **Other** from the categories on the left, select **SolarWinds Service Desk**, and then select **Connect**.
+1. In Power BI Desktop, select **Get Data** from **Home**. Select **SolarWinds Service Desk**, and then select **Connect**.
 
    ![Screenshot of Get Data from SolarWinds Service Desk.](./media/solarwinds-service-desk/get-data.png)
 
-2. Sign in with the JSON web token you generated as described in prerequisites, and then select **Connect** to verify your access to SolarWinds Service Desk.
+2. Sign in with the JSON web token you generated as described in prerequisites, and then click **Connect** to verify your access to SWSD.
 
    ![Screenshot of Auth Key Dialog.](./media/solarwinds-service-desk/auth-key.png)
 
-3. In the **Navigator** dialog box, select the table you want to import. You can then either load or transform the data.
+3. In the **Navigator** dialog box, select the tables you want to import. and then click **Load**.
 
-   ![Screenshot of Get Data from SolarWinds Service Desk Navigator.](./media/solarwinds-service-desk/nav-data.png)
+   ![Screenshot of SolarWinds Service Desk Navigator.](./media/solarwinds-service-desk/nav-data.png)
 
-## Limitations and issues
+4. After completion of the steps above, you should see your chosen models load under Data view. The records you have just imported were created in SWSD during last month.
```

Comment:
Change "After completion of the steps above, you..." to "After you complete the previous steps, you..." to meet Microsoft style rules.